server/controllers/manage/residentProfileController.ts

```diff
@@ -42,10 +52,40 @@ export default class ResidentProfileController {
           ])
           subHeading = 'Offence and sentence'
           cardList = offencesCards(offences, offenceAnswers)
+          sectionTemplate = 'manage/resident/partials/cardList.njk'
         }
         if (subTab === 'licence') {
           subHeading = 'Licence'
           cardList = licenseCards()
+          sectionTemplate = 'manage/resident/partials/cardList.njk'
+        }
+      }
+
+      if (activeTab === 'placement') {
+        const subTab = section as PlacementSubTab
+        sideNavigation = placementSideNavigation(subTab, crn, placementId)
+
+        switch (subTab) {
+          case 'placement-details':
+            subHeading = 'Placement details'
+            cardList = placementDetailsCards()
+            sectionTemplate = 'manage/resident/partials/cardList.njk'
+            break
+          case 'application':
+            subHeading = 'Application'
+            cardList = applicationCards()
+            sectionTemplate = 'manage/resident/partials/cardList.njk'
+            break
+          case 'previous-ap-stays':
+            subHeading = 'Previous AP stays'
+            {
+              const spaceBookings = await this.personService.getSpaceBookings(req.user.token, crn, includeCancelled)
+              cardList = previousApStaysCards(spaceBookings)
+              sectionTemplate = 'manage/resident/partials/cardList.njk'
+            }
+            break
+          default:
+            break
```

Comment:
It's fine for now, but I'm thinking that most of this logic needs shipping out, or this controller is going to become massive. 
Also, we could really do with a way to run all the API calls in parallel and that's quite a complex thing to achieve structurally.